site/plans.html.jet

```diff
@@ -59,9 +66,24 @@
                   {{ if p.Page }}
                     <a class="plan-page-link btn btn-light text-dark btn-block" href="{{ routeToSlug(p.Page.Slug) }}">{{i18n("plan_read_more")}}</a>
                   {{ end }}
-                  <s72-pricing-buttons class="btn-block" data-slug="{{p.Slug}}" data-title="{{p.Name}}" />
+                  {{ if p.PlanType == "free" }}
+                    {{ if p.HasExpiryDate() }}
```

Comment:
I hesitate to suggest this, but just with the repeated blocks of `<s72-user-anon><a></s72-user-anon>` you _could_ potentially not check `p.HasExpiryDate()` and use a date variable that is either the plan expiry or a future date and then you could always use `<s72-before>`.

E.g. pseudo-code:

```html
{{expiry := p.HasExpiryDate() ? p.ExpiryDate : [tomorrow, next week, max date - dealers choice]}}
{{if p.PlanType == "free"}}
  <s72-before datetime="{{expiry}}">
    <s72-user-anon>
      <a>
    </s72-user-anon>
  </s72-before>
{{ else }}
  <s72-pricing-buttons>
{{ end }}
```

Reply:
Might leave this as is; a line like

```
{{ expiry := plan.HasExpiryDate() ? plan.ExpiryDate.UnixMilli() : 4100670000000 }}
```

is borderline hackery even for me

Reply:
Well, when you put it like _that_ sure lol

I was thinking more `time.Now().AddDate(1, 0, 0)` or similar.